src/components/FormFieldComponent.vue

```diff
@@ -261,6 +258,13 @@
         return defaultNoOptionsMessage
       }
     },
+    watch: {
+      pending (isPending) {
+        if (!isPending) {
```

Comment:
maybe add note that were actually watching for the flip of pending, this is not directly apparent when looking at the code i think